zephyr/prj.conf

```diff
@@ -34,6 +34,6 @@ CONFIG_LOG_MODE_MINIMAL=y
 
 # ThingSet protocol interface at UART serial
 CONFIG_THINGSET_SERIAL=y
-
+CONFIG_UEXT_SERIAL_THINGSET=y
```

Comment:
If we decide to change this default we should at least put it into a dedicated commit. Need to think twice if it's a good idea.... probably yes.